utils/check_dashboard_name_uniqueness.ts

```diff
@@ -22,21 +22,40 @@ const findMatchingDashboardNames = (
 };
 
 const main = () => {
-  const allDashboards = Dashboard.getAll();
-  const nameMatches = findMatchingDashboardNames(allDashboards);
-  console.log(nameMatches);
-
-  if (nameMatches.length > 0) {
-    console.error(`ERROR: Found matching quickstart dashboard names`);
-    console.error(`Punctuation and white space are removed before comparison`);
-    nameMatches.forEach((m) =>
-      console.error(`${m.identifier} in ${removeRepoPathPrefix(m.configPath)}`)
-    );
-    console.error(
-      `Please update your quickstart dashboard's name to be unique\n`
-    );
+  const catchErrors = []
+  try {
```

Comment:
Is this try/catch block even needed here anymore? It doesn't look like any of these methods throw an error.